lib/matplotlib/backends/backend_agg.py

```diff
@@ -625,13 +625,30 @@ def print_jpg(self, filename_or_obj, *args, **kwargs):
 
         # add TIFF support
         def print_tif(self, filename_or_obj, *args, **kwargs):
+            """
+            Other Parameters
+            ----------------
+            compressed : bool
+                If present, indicates that this image
+                should be stored using the LZW compression algorithm.
+            """
             buf, size = self.print_to_buffer()
             if kwargs.pop("dryrun", False):
                 return
             image = Image.frombuffer('RGBA', size, buf, 'raw', 'RGBA', 0, 1)
             dpi = (self.figure.dpi, self.figure.dpi)
-            return image.save(filename_or_obj, format='tiff',
-                              dpi=dpi)
+            #add TIFF compression support
+            compressed = kwargs.pop("compressed", False)
+            if compressed:
+                libtiff_original_value = TiffImagePlugin.WRITE_LIBTIFF
```

Comment:
@story645 I'm having some issues coming up with some meaningful tests since we now do a pull pass-through of the parameters to the backend. Can you suggest some tests, if needed?